package.json

```diff
@@ -34,6 +34,8 @@
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/preset-typescript": "^7.9.0",
+    "@commitlint/cli": "^13.1.0",
+    "@commitlint/config-conventional": "^13.1.0",
```

Comment:
I ran `npm-check-updates` on core, and looks like we're way behind on v13. Did you try v17? I think it would be great to update core to that version if it isn't too breaking.

```
 @commitlint/cli                                   ^13.1.0  →       ^17.1.2
 @commitlint/config-conventional                   ^13.1.0  →       ^17.1.0
```

Reply:
🤯 I could have sworn i had checked for the most recent versions. I'll try that out now!

Reply:
@mcoker done!